src/shared/codelens/codeLensUtils.ts

```diff
@@ -123,6 +126,81 @@ function makeAddCodeSamDebugCodeLens(
     return new vscode.CodeLens(range, command)
 }
 
+/**
+ * Tied to the AWS.addSamDebugConfig command: lets a user create a config tied to a handler via command instead of codelens
+ * Renders a quick pick using the first 200 characters of the first line of the funciton declaration + function line number
+ * @param document Curr document
+ * @param lenses Codelenses returned via CodeLensProvider, which we extract the information from
+ */
+export async function invokeCodeLensCommandPalette(
+    document: Pick<vscode.TextDocument, 'getText'>,
+    lenses: vscode.CodeLens[]
+): Promise<void> {
+    const handlers: (vscode.QuickPickItem & { lens?: vscode.CodeLens })[] = lenses
+        .filter(lens => {
+            // remove codelenses that go to the invoker UI
+            // maybe move this into the workflow at some point (drop down to one)
+            return (
+                lens &&
+                lens.command &&
+                lens.command.arguments &&
+                lens.command.arguments.length === 3 &&
+                lens.command.arguments![2] !== true
+            )
+        })
+        .map(lens => {
+            return {
+                // lens is currently pulling the entire function, not just the declaration
+                label: document.getText(
+                    new vscode.Range(
+                        lens.range.start,
+                        new vscode.Position(lens.range.start.line, lens.range.start.character + 200)
+                    )
+                ),
+                detail: localize(
+                    'AWS.pickDebugHandler.range',
+                    'Function on line {0}',
+                    (lens.range.start.line + 1).toString()
+                ),
+                lens,
+            }
+        })
+    if (handlers.length === 0) {
+        handlers.push({
+            label: 'No handlers found in current file',
+            detail: 'Please ensure your language extension is working',
+            description: 'Click to go back',
+        })
```

Comment:
Do these need to be localized?

Reply:
Beat you to it https://github.com/aws/aws-toolkit-vscode/commit/d10592878e6570351fc41604b903bd4d215fc072

Reply:
Yup haha

Reply:
Ack NVM forgot about this one 💀